src/stylesheets/components/_banner.scss

```diff
@@ -1,160 +1,171 @@
+$banner-font-family: 'sans';
+
 .usa-banner {
-  background-color: $color-gray-lightest;
-  padding-bottom: 0.7rem;
+  @include u-bg($base-lighter);
+  @include u-font-family($banner-font-family);
+  @include u-padding-bottom(1);
+  @include u-position('relative');
 
-  @include media($small-screen) {
-    font-size: $h6-font-size;
-    padding-bottom: 0;
+  @include at-media('mobile-lg') {
+    @include u-font-size($banner-font-family, $theme-h6-font-size);
+    @include u-padding-bottom(0);
   }
 }
 
 .usa-banner-content {
-  @include padding(null $site-margins-mobile 3px $site-margins-mobile);
-  background-color: transparent;
-  font-size: $h5-font-size;
-  padding-top: 2rem;
-  width: 100%;
+  @include u-bg('transparent');
+  @include u-padding-bottom(2px);
+  @include u-padding-top(2.5);
+  @include u-padding-x($site-margins-mobile);
+  @include u-width('full');
 
-  @include media($medium-screen) {
-    padding-bottom: 2.3rem;
-    padding-top: 4rem;
+  @include at-media('tablet') {
+    @include u-padding-bottom(3);
+    @include u-padding-top(5);
   }
 
-  @include media($nav-width) {
-    @include padding(null $site-margins);
+  @include at-media($theme-nav-width) {
+    @include u-padding-x($site-margins);
   }
 
   p {
+    @include u-font-size($banner-font-family, $theme-h5-font-size);
+    @include u-line-height($banner-font-family, $theme-base-line-height);
+
     &:first-child {
-      margin-top: 1rem;
+      @include u-margin-top(1);
 
-      @include media($medium-screen) {
-        margin-top: 0;
+      @include at-media('tablet') {
+        @include u-margin-top(0);
       }
     }
   }
 }
 
 .usa-banner-inner {
-  @include outer-container($site-max-width);
-  @include padding(null $site-margins-mobile);
+  @include grid-container($theme-site-max-width);
+  @include u-padding-x($theme-site-margins-mobile);
```

Comment:
Why does this use `$theme-site-margins-mobile` and https://github.com/uswds/uswds/pull/2668/files#diff-f387466a429ac4e7ace4438ebb06393bR19 and https://github.com/uswds/uswds/pull/2668/files#diff-d44185fbb54964385b1a02ae836e77efR8 uses `$site-margins-mobile`? 

When would you use `$theme-site-margins-mobile` over `$site-margins-mobile` and vice versa?

Reply:
In general, I was trying to scope the work in this PR as much as possible to this specific component. `$theme-site-margins-mobile` should be the new variable going forward, used anywhere `$site-margins-mobile` is used. We could begin to globally replace the spacing unit variables now, I suppose, but I started to go in that direction and found myself in deeper, broader PR water in my initial PR. 

Spacing units would be a good candidate for global replacement (in most instances) — something like `font-size` variables is definitely a bigger job and one I'd like to save for (possibly) a PR of its own or (more likely) something that's only integrated piece by piece.

Reply:
OK, I changed the vars in the old grid to use the new margins theme vars